collectors/carbonblack/test/utils_test.js

```diff
@@ -115,5 +115,104 @@ describe('Unit Tests', function () {
+            url: "url",
+            method: "GET",
+            requestBody:"",
+            typeIdPaths: [{ path: ["eventId"] }],
+            tsPaths: [{ path: ["eventTime"] }]
+        };
+        let accumulator = [];
+        const apiEndpoint = process.env.paws_endpoint;
+        const clientSecret = process.env.paws_api_secret;
+        const clientId = process.env.paws_api_client_id;
+
+        utils.getAPILogs(apiDetails, accumulator, apiEndpoint, state, clientSecret, clientId, maxPagesPerInvocation).catch(err => {
+            assert.equal(err.message, "Failed to fetch API logs due to an authentication issue", "Error message is not correct");
+            alserviceStub.get.restore();
+            done();
+        });
+    });
+});
+
+
+describe('Get API Logs (POST Request) with Error', function () {
+    it('Get API Logs with Error (POST)', function (done) {
+        alserviceStub.post = sinon.stub(RestServiceClient.prototype, 'post').callsFake(
+            function fakeFn(path, extraOptions) {
+                return new Promise(function (resolve, reject) {
+                    return reject(new Error("Failed to fetch API logs due to an authentication issue"));
+                });
+            });
+        let maxPagesPerInvocation = 5;
+        const startDate = moment().subtract(5, 'minutes');
+        let state = {
+            stream: "SearchAlerts",
+            since: startDate.toISOString(),
+            until: startDate.add(5, 'minutes').toISOString(),
+            poll_interval_sec: 1
+        };
+        let apiDetails = {
+            url: "url",
+            method: "POST",
+            requestBody:{
+                "criteria": {
+                    "create_time": {
+                        "end": state.until,
+                        "start": state.since
+                    },
+                },
+                "rows": 0,
+                "start": 0
+            },
+            typeIdPaths: [{ path: ["id"] }],
+            tsPaths: [{ path: ["last_update_time"] }]
+        };
+        let accumulator = [];
+        const apiEndpoint = process.env.paws_endpoint;
+        const clientSecret = process.env.paws_api_secret;
+        const clientId = process.env.paws_api_client_id;
 
+        utils.getAPILogs(apiDetails, accumulator, apiEndpoint, state, clientSecret, clientId, maxPagesPerInvocation).catch(err => {
+            assert.equal(err.message, "Failed to fetch API logs due to an authentication issue", "Error message is not correct");
+            alserviceStub.post.restore();
+            done();
+        });
+    });
+});
 
+describe('Get API Details when state.stream is null', function () {
+    it('Get API Details when state.stream is null', function (done) {
+        const startDate = moment().subtract(5, 'minutes');
+        const orgKey = "orgKey";
+        let apiDetails = [];
+        const apiNames = JSON.parse(process.env.collector_streams_null);
+        apiNames.map(stream => {
```

Comment:
process.env.collector_streams_null pls rename to proper name

Reply:
pls check the above comment and resolve

Reply:
I have added the reason, why I have added this code, in this comment. (https://github.com/alertlogic/paws-collector/pull/318#discussion_r1108187632)

Reply:
rename the variable name collector_streams_null